addons/event/models/event_event.py

```diff
@@ -10,7 +10,7 @@
 from odoo.addons.base.models.res_partner import _tz_get
 from odoo.exceptions import UserError, ValidationError
 from odoo.osv import expression
-from odoo.tools import format_datetime, is_html_empty
+from odoo.tools import format_datetime, is_html_empty, html2plaintext
```

Comment:
@thsh-odoo Simply rearrange it in alphabetical order, and everything else looks good to proceed.